styles/index.css

```diff
@@ -0,0 +1,71 @@
+* {
+    /* border: 1px solid black; */
+    color: #2F3C7E;  
+}
+
+::selection {
+    background-color: #2F3C7E;
+    color: #f6edee;
+}
```

Comment:
Consider placing shared styles in a shared css file that all pages include, an put only styles specific to the page in the per-page stylesheets.